### PR TITLE
feat: add support theme for CarouselCard, CarouselImage, CheckboxGroup, CheckboxToggle

### PR DESCRIPTION
**Pull request by wildergd**
fix: #1269
#1172 

## Changes proposed in this PR:
- add theme support to CarouselCard component
- add theme support to CarouselImage component
- add theme support to CheckboxGroup component
- add theme support to CheckboxToggle component

- [x] I have followed (at least) the [PR section of the contributing guide](https://github.com/nexxtway/react-rainbow/blob/master/CONTRIBUTING.md#submitting-a-pull-request).

@nexxtway/react-rainbow

### CODE REVIEW

**Comment by wildergd** [comment]
verificar... he encontrado muchas veces este color #dcdfe7 fijo en bordes. no es un color de los predefinidos... no se si esto se migra o se deja asi...

**Comment by wildergd** [comment]
este shadow no esta en los predefinidos... box-shadow: 0 0 1px 0 ${COLOR_GRAY_2};
hay uno igual pero con COLOR_GRAY_3...  pongo ese?

File: src/components/CarouselCard/styled/imagesUl.js [hunk 1]
```diff
@@ -1,21 +1,16 @@
...
-    return { shadowBrand2: `0 0 8px ${brandMainColor}` };
-})`
+const StyledImagesUl = attachThemeAttrs(styled.ul)`
     width: 100%;
     height: 100%;
     overflow: hidden;
     position: relative;
     display: flex;
     border-radius: ${BORDER_RADIUS_1};
     border: 1px solid #dddbda;
```
Comment (TahimiLeonBravo) [inline src/components/CarouselCard/styled/imagesUl.js]: We need to add customize the `border`

File: src/components/CarouselImage/styled/innerContainer.js [hunk 2]
```diff
@@ -30,7 +26,7 @@ const StyledInnerContainer = styled.div.attrs(props => {
         :focus {
             color: inherit;
             text-decoration: none;
-            border: 1px solid ${props.brandMainColor};
+            border: 1px solid ${props.palette.border.divider};
```
Comment (LeandroTorresSicilia) [inline src/components/CarouselImage/styled/innerContainer.js]: Why change this brand color?

File: src/styles/defaultTheme.js [hunk 3]
```diff
@@ -46,6 +46,7 @@ export default {
         },
         border: {
             main: COLOR_GRAY_3,
+            secondary: COLOR_GRAY_2,
             divider: COLOR_GRAY_2,
             disabled: COLOR_GRAY_2,
```
Comment (LeandroTorresSicilia) [inline src/styles/defaultTheme.js]: @TahimiLeonBravo  check this three have the same color

File: src/styles/helpers/normalizeTheme.js [hunk 4]
```diff
@@ -51,6 +51,7 @@ function resolveCustomBackground(background) {
             },
             border: {
                 main: dark ? lighten(background, 0.3) : darken(background, 0.3),
+                secondary: dark ? lighten(background, 0.2) : darken(background, 0.2),
                 divider: dark ? lighten(background, 0.1) : darken(background, 0.1),
                 disabled: dark ? lighten(background, 0.1) : darken(background, 0.1),
```
Comment (LeandroTorresSicilia) [inline src/styles/helpers/normalizeTheme.js]: I see they are equals when default but computed values are differents

**Pull request merged**